archive/2022-11-15-DssSpell/Goerli-DssSpell.sol

```diff
@@ -0,0 +1,131 @@
+// SPDX-FileCopyrightText: © 2020 Dai Foundation <www.daifoundation.org>
```

Comment:
the archive date should match cast date (2022-11-14), please redo as follow:
`make archive-spell date=2022-11-14`